news/9191.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Handle case of list default values in UpdatingDefaultsHelpFormatter
+Happens because we pass a list from --extra-index-url
```

Comment:
```suggestion
Fix crash when logic for redacting authentication information from URLs
in ``--help`` is given a list of strings, instead of a single string.
```